utils/CMakeLists.txt

```diff
@@ -31,3 +31,4 @@ add_custom_target(OMONNXCheckVersion
 
 # Scan files for supported ops
 add_onnx_mlir_supported_ops(${ONNX_MLIR_SRC_ROOT}/test/backend/inference_backend.py cpu)
+add_onnx_mlir_supported_ops(${ONNX_MLIR_SRC_ROOT}/test/accelerators/NNPA/backend/CMakeLists.txt NNPA)
```

Comment:
@AlexandreEichenberger My understanding is that these documents are not generated by the normal build commands for either CPU or NNPA. To generate these documents, we need to manually run `make --build . --config Release --target onnx_mlir_supported_ops`. So I think it is OK to put it here.
I have confirmed that moving it in `test/accelerators/NNPA/backend/CMakeLists.txt` works, but it might be better in the `utils` directory.